kickstarts/base.ks.erb

```diff
@@ -1,5 +1,8 @@
 # Kickstart file automatically generated by anaconda.
-# Run the text based installer to see dependency info in screenshots
+
+# By default run the text based installer to see dependency info in screenshots
+<% if ENV['KS_GUI'] == '1' then "" else "text" end %>
+
```

Comment:
Hi @geertj thanks for making this change, instead of ENV variable, I'd prefer if this was a vmbuild.rb option (as defined in cli.rb), introducing --text  and --graphics options.  If you prefer we can do this in another PR.